TVnMoviesApp/Controller/Home/HomeViewController.swift

```diff
@@ -15,9 +17,11 @@ class HomeViewController: UIViewController {
     @IBOutlet weak var activityLoading: UIActivityIndicatorView!
     
     var manager = Manager()
-    var resultsArray: [ResultOfSearch] = []
+    let disposeBag = DisposeBag()
+    let resultsArray = Variable<[ResultOfSearch]>([])
```

Comment:
BehaviorRelay works pretty much the same was as Variable. Variable is deprecated and BehaviorRelay is replacing it.

You could also use Variable/BehaviorRelay to call reloadData on collectionView. You could subscribe to it and when it emits, reloadData. However in this case it would be overkill, cause you already know when resultsArray is updated, cause you do it here, in the same class. You can just call reloadData directly after updating resultsArray.